README.md

```diff
@@ -1 +1,107 @@
-# Blueprint
\ No newline at end of file
+# Blueprint
+
+aka **How We Collaborate** :zap:
+
+##Contents
+
+[Overview](#overview)  
+[Why Process?](#why-process)  
+[Team](#team)  
+[Workflow](#workflow)  
+[Sprints](#sprints)  
+[Notes](#notes)  
+[Contributors](#contributors)  
+
+##Overview
+
+This document is an overview of the [Vimeo Mobile and TV Group's](#notes)<sup>1</sup> product development process (a process based on the [scrum](#notes)<sup>2</sup> and [agile](#notes)<sup>3</sup> methodologies). It describes [why process](#why-process) is important and outlines three aspects of our process: [team](#team), [workflow](#workflow), and [sprints](#sprints). This document focuses on high-level guidelines rather than specific implementation tactics in order to leave room for teams to apply them as they see fit. 
+
+##Why Process?
+
+Process helps us to:
+ 
+1. Produce high quality work at a predictable [pace](#notes)<sup>4</sup>, 
+3. Promote the development of an extraordinary team.
+ 
+##Team
+ 
+A product team is a co-located, cross-functional group of roughly 4-7 people who collaborate to bring ideas to life. Proximity builds relationships. Cross-functionality fosters autonomy and self-sufficiency. Limiting team size promotes ownership.
+ 
+![Team composition](https://github.com/vimeo/Blueprint/blob/working-draft/assets/team-composition.png)    
+<sup>*Team composition*</sup>
+
+Teams are externally trusted, transparent, and accountable. Teams are internally creative, balanced, and practical. 
+
+The goal of a team is to produce meaningful, high quality work at a predictable [pace](#notes)<sup>4</sup>.
+ 
+##Workflow
+ 
+Workflow is a structured approach to how a team evolves an idea from inception to production. Teams evolve each idea one function at a time, with each function’s output being the next function’s input. 
+
+![Workflow and deliverables](https://github.com/vimeo/Blueprint/blob/working-draft/assets/workflow.png)    
+<sup>*Workflow and deliverables*</sup>
```

Comment:
Agreed, I think that's something that we can speak to if someone has questions about the doc, I'm also going to add some info addressing this per @nicolelehrer and @howardrigberg 's comment above